rasa/core/tracker_store.py

```diff
@@ -732,7 +809,15 @@ def session_scope(self):
         """Provide a transactional scope around a series of operations."""
         session = self.sessionmaker()
         try:
+            ensure_schema_exists(session)
             yield session
+        except ValueError as e:
+            rasa_cli_utils.print_error_and_exit(
+                f"Requested PostgreSQL schema '{e}' was not found in the database. To "
```

Comment:
but `str(e)` is already exactly that because of 
```python
raise ValueError(schema_name)
```
in `ensure_schema_exists()` (and `schema_name = os.environ.get(POSTGRESQL_SCHEMA)`)

Reply:
oh ok! jist not very intuitive 😀